src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -502,7 +502,7 @@ protected function initializeBundles()
      */
     protected function getContainerClass()
     {
-        return $this->name.ucfirst($this->environment).($this->debug ? 'Debug' : '').'ProjectContainer';
+        return preg_replace('/(^[0-9]*|[^a-zA-Z0-9_]+)/', '', $this->name.ucfirst($this->environment)).($this->debug ? 'Debug' : '').'ProjectContainer';
```

Comment:
If I have the following 3 applications, will it still work after this change? Wouldn't it be better to just prefix something?
 - `1app`
 - `2app`
 - `3app`